pyteal/ast/if_.py

```diff
@@ -63,4 +69,27 @@ def __str__(self):
     def type_of(self):
         return self.thenBranch.type_of()
```

Comment:
It might be worthwhile to raise the same error here as in `__teal__` if there is no `thenBranch`.